packages/rest-typings/src/v1/oauthapps/UpdateOAuthAppParamsPOST.ts

```diff
@@ -0,0 +1,40 @@
+import { ajv } from '../Ajv';
+
+export type UpdateOAuthAppParams = {
+	appId: string;
+	name: string;
+	active: boolean;
+	clientId?: string | undefined;
+	clientSecret?: string | undefined;
+	redirectUri: string;
+};
+
+const UpdateOAuthAppParamsSchema = {
+	type: 'object',
+	properties: {
+		appId: {
+			type: 'string',
+		},
+		name: {
+			type: 'string',
+		},
+		active: {
+			type: 'boolean',
+		},
+		clientId: {
+			type: 'string',
+			nullable: true,
+		},
+		clientSecret: {
+			type: 'string',
+			nullable: true,
+		},
```

Comment:
```suggestion
```
I guess we shouldn't include these params (not even as optional) since they can never be updated by the user.